tests/input/sync_synonym/mondo-excluded-synonyms.tsv

```diff
@@ -0,0 +1,13 @@
+?mondo_id	?synonym	?hasDbXref_xref	?source_xref
```

Comment:
This file will be replaced by a "hard-coded" file that @matentzn will create that can be used directly in the mondo-ingest repo without needing to use `mondo-edit.obo` in the synonym sync pipeline to extract the excluded synonyms or handle the dubious ownership issue that has come up.

Reply:
Completed in: 44c978fe2c5e4fc362d12d451b1ed8e3386146f3